wordle-web/components/hamburger-menu.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-card>
+    <v-btn icon @click="toggleDialog">
+        <v-icon color="secondary"> mdi-hamburger </v-icon>
+    </v-btn>
+    <v-dialog
```

Comment:
I probably would suggest putting this in a navigation drawer or in a v-menu, but what you did isn't wrong. Good job!